### ISSUE TITLE
Add ability to create a Cassandra cluster extended on multiple k8s clusters

### ISSUE THREAD

**Issue by cscetbon** [datastax/cass-operator#319]
Another ticket to be CassKop compatible. This ticket is to have something similar to [multiCassKop](https://orange-opensource.github.io/casskop/docs/2_setup/3_multi_casskop). So it's different from #318 as it is about extending a cluster on multiple kubernetes clusters. Of course it also needs to work with multiple datacenters, so it would probably make sense to implement it after #318.

**Comment by jdonenine** [comment]
Migrated to https://github.com/k8ssandra/k8ssandra/issues/943
